src/app/page.tsx

```
@@ -0,0 +1,14 @@
+export default function HomePage() {
+  return (
+    <main className="p-8">
+      <h1 className="text-3xl font-bold">BrandBoard</h1>
+      <p className="text-muted-foreground mt-2">
+        Foundation established. Sign in to access your dashboard.
+      </p>
+      <div className="mt-6 flex gap-4">
+        <a className="px-4 py-2 bg-brand-600 rounded text-white" href="/sign-in">Sign In</a>
```

ESLint

[error] Do not use an `<a>` element to navigate to `/sign-in/`. Use `<Link />` from `next/link` instead. See: https://nextjs.org/docs/messages/no-html-link-for-pages

(@next/next/no-html-link-for-pages)